.github/workflows/snyk.yml

```diff
@@ -0,0 +1,20 @@
+name: Snyk Security Scan
+
+on:
+  pull_request:
+    branches: [ main ]
```

Comment:
how does snyk determine the version it is pushing the report for? Will these be reported for main or will we have a snyk entry for each branch (that we'll have to clean up later)

Reply:
approved as I'm fine if we find this out after the action is merged, but we should take a look before we consider this fully done